server/apps/publish/archive_publish.py

```diff
@@ -1032,7 +1074,12 @@ def get_subscribers(self, doc, target_media_type):
             # step 2
             if not self.takes_package_service.get_take_package_id(doc):
                 # Step 3
-                active_subscribers = get_resource_service('subscribers').get(req=None, lookup={'is_active': True})
+                if doc.get(EMBARGO) and doc.get(EMBARGO) > utcnow():
+                    active_subscribers = list(get_resource_service('subscribers').get(
+                        req=None, lookup={'subscriber_type': SUBSCRIBER_TYPES.WIRE, 'is_active': True}))
+                else:
+                    active_subscribers = get_resource_service('subscribers').get(req=None, lookup={'is_active': True})
```

Comment:
Similar suggestion here - we could merge both branches and just add the `subscriber_type` condition to the lookup dict if necessary.